src/util/render/backgrounds/index.ts

```diff
@@ -69,3 +63,18 @@ const drawRuled = (ctx: CanvasRenderingContext2D, meta: PageMeta): void => {
     ctx.moveTo((1 - boundary) * meta.size.width, 0)
     ctx.lineTo((1 - boundary) * meta.size.width, meta.size.height)
 }
+
+const drawDocument = async (
+    ctx: CanvasRenderingContext2D,
+    meta: PageMeta
+): Promise<void> => {
+    const { attachId, documentPageNum } = meta.background
+
+    if (attachId === undefined || documentPageNum === undefined) return
+
+    const attachment = board.getState().attachments[attachId] as PDFAttachment
```

Comment:
Maybe we should separate the pdf attachements from the rest in the state for more robust typing ? 🤔 They aren't bound to pages the same way as the other attachments will be 🤔

Reply:
well, since this is function is only concerned about rendering the page background, I think it is fine to explicitly use PDFAttachment